src/SURFnet/OATHBundle/DependencyInjection/SURFnetOATHExtension.php

```diff
@@ -19,14 +19,13 @@ class SURFnetOATHExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
-        $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        //$configuration = new Configuration();
+        //$config = $this->processConfiguration($configuration, $configs);
 
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $storage = $container->getParameter('surfnet_oath.userstorage');
```

Comment:
Creating a configuration array in the parameters.yml kind of feels like a hack. You are now unable to utilize the built in configuration validation. Which could result in slightly harder to debug configuration issues. 

I know this code is reused, but this could be improved on in the future.